docs/upgrade/0.6.x_to_0.7.rst

```diff
@@ -0,0 +1,104 @@
+Upgrade from 0.6.x to 0.7.x
+===========================
+
+Updating the Tails workstations
+-------------------------------
+
+All Tails drives should be updated to Tails 3.7, which is scheduled to be 
+released on May 9, 2018. Follow the graphical prompts on your workstations to
+perform this upgrade.
+
+For the *Journalist Workstations* and the *Admin Workstation*, you will also
+need to update the SecureDrop code. First, ensure that you have 
```

Comment:
To make a little clearer what this is and why it's important, maybe we could say something like:

`SecureDrop code` -> `SecureDrop code (this configures the Workstation environment, for example the SecureDrop shortcuts on the Tails Desktop)`